pkg/plugin/test-keeper/metrics.go

```diff
@@ -0,0 +1,72 @@
+package testkeeper
+
+import (
+	"github.com/arquillian/ike-prow-plugins/pkg/log"
+	gogh "github.com/google/go-github/github"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/arquillian/ike-prow-plugins/pkg/server"
+)
+
+const (
+	// WithTests is a label used in prometheus metrics pull_requests_total for pull request containing some tests.
+	WithTests = "with_tests"
+	// WithoutTests is a label used in prometheus metrics pull_requests_total for pull request without any tests.
+	WithoutTests = "without_tests"
+)
+
+var (
+	pullRequestsCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "pull_requests_total",
+		Help: "Total number of pull requests.",
+	}, []string{"full_name", "type"})
```

Comment:
I'd like to welcome some information about the plugin if it is plugin-specific metric

Reply:
I think you can select host in PCP and graphana charts due to which I believe this is not needed

Reply:
what if I don't have it filtered (when I display all graphs) - then it is visible as well, right? Then it would make sense adding the plugin name to the metric name and its description